src/test/java/cz/czechitas/AdminLoginPage.java

```diff
@@ -0,0 +1,39 @@
+package cz.czechitas;
+
+import java.util.*;
+import org.openqa.selenium.*;
+import org.openqa.selenium.support.ui.*;
+
+public class AdminLoginPage {
+
+    private final By loginUsernameSelector = By.id("username");
+    private final By loginPasswordSelector = By.id("password");
+    private final By loginButtonSelector = By.id("formSubmitButton");
+    private final By errorAdminLoginSelector = By.className("alert-error");
+    private final By adminPageSelector = By.className("icon-user");
+    private final WebDriver driver;
+
+    public AdminLoginPage(WebDriver driver) {
+        this.driver = driver;
+
+    }
+
+    public void openPage() throws InterruptedException {
+        driver.navigate().to("http://czechitas-shopizer.azurewebsites.net/admin/logon.html");
+        Thread.sleep(5000);
```

Comment:
Davat sleep do openPage() neni dobry napad, takzdy jeden test to protahne od 5 vterin. Predtav si, ze mas testu 200.